src/test/kotlin/no/skatteetaten/aurora/boober/feature/AlertsFeatureTest.kt

```diff
@@ -154,6 +182,30 @@ class AlertsFeatureTest : AbstractFeatureTest() {
         assertThat(generatedResources).hasSize(1)
         val alert = (generatedResources[0].resource as Alerts).spec.alert
         assertThat(alert.delay).isEqualTo("2")
-        assertThat(alert.connection).isEqualTo("en-annen-koblingsregel")
+        assertThat(alert.connections).containsOnly("en-annen-koblingsregel")
+    }
+
+    @Test
+    fun `should fail validation when legacy connection and new connections is defined`() {
+        assertThat {
+            createAuroraDeploymentContext(
+                """{
+                  "alertsDefaults": {
+                    "connections": [
+                      "aurora-email"
+                    ]
+                  },
+                  "alerts": {
+                    "is-down": {
+                      "enabled": true,
+                      "delay": 1,
+                      "expr": "test",
+                      "connection": "aurora-mattermost",
+                      "severity": "warning"
+                    }
+                  }
+                }"""
+            )
+        }.singleApplicationErrorResult(AlertsFeature.Errors.InvalidLegacyConnectionAndConnectionsProp.message)
```

Comment:
Kan vi ha en test for containsDeprecatedConnection. Noe ala dette:
```kotlin
    @Test
    fun `should return warnings`() {
        val spec = createAuroraDeploymentSpecForFeature(
            """{
                  "alerts": {
                    "is-down": {
                      "enabled": true,
                      "delay": 1,
                      "expr": "test",
                      "connection": "aurora-mattermost",
                      "severity": "warning"
                    }
                  }
                }"""
        )
        val alertsFeature = feature as AlertsFeature
        val containsDeprecatedConnection = alertsFeature.containsDeprecatedConnection(spec)
        assertThat(containsDeprecatedConnection).isTrue()
    }

```

Vi kan også ha en her at den returnerer false hvis alert ikke er konfigurert eller connections brukes. 
Vet at noe av det er testet i deployfacadetest, men det dekker ikke hele.

Reply:
la til foreslått test, og tester for begge scenarioene